src/main/rules/GCI1442/GCI1442.json

```diff
@@ -0,0 +1,16 @@
+{
+  "title": "Reduce memory footprint of dataclasses",
+  "type": "CODE_SMELL",
+  "status": "ready",
+  "remediation": {
+    "func": "Constant\/Issue",
+    "constantCost": "1min"
+  },
+  "tags": [
+    "memory",
+    "python",
```

Comment:
please delete python tag.
no need python tag because if only for python, this rule only will be implemented inside python plugin.